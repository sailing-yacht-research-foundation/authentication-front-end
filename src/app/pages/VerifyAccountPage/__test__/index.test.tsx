import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { VerifyAccountPage } from '..';

const shallowRenderer = createRenderer();

describe('<VerifyAccountPage />', () => {
  it('should render and matches the snaphshot', () => {
    shallowRenderer.render(<VerifyAccountPage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});