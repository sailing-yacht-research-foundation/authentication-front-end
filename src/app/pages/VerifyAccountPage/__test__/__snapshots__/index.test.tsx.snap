// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VerifyAccountPage /> should render and matches the snaphshot 1`] = `
<Row
  align="middle"
  justify="center"
  style={
    Object {
      "minHeight": "100vh",
      "padding": "0 15px",
    }
  }
>
  <VerifyAccountForm />
</Row>
`;
